examples/Bunch/include/simulation_defines/param/fieldBackground.param

```diff
@@ -81,7 +81,7 @@ namespace picongpu
             /* example 1: TWTS background pulse */
             /** unit: meter */
             const double WAVE_LENGTH_SI = 0.8e-6;
-            
+
             /** UNITCONV */
             /* const double UNITCONV_Intens_to_A0 = SI::ELECTRON_CHARGE_SI
                 * SI::ELECTRON_CHARGE_SI * 2.0 * WAVE_LENGTH_SI * WAVE_LENGTH_SI / (4.0 * PI * PI
```

Comment:
at this point, it's a C++ limitation that converting on the fly is not as easy as it could be with C++11.
therefore, it is not *dead code* but *examples* how to configure lasers in units of `a0`, in terms of `intensity` or in terms of the `electric field strength`.